src/langs/en.json

```diff
@@ -101,7 +101,15 @@
   "INVITATIONS_TABLE_EMAIL_HEADER": "Email",
   "INVITATIONS_TABLE_INVITATION_HEADER": "Invitation",
   "INVITATIONS_TABLE_PERMISSION_HEADER": "Permission",
-  "ITEM_CATEGORIES_SELECTION_TITLE": "Category",
+  "ITEM_CATEGORIES_CONTAINER_TITLE": "Category",
+  "ITEM_CATEGORIES_CONTAINER_MISSING_WARNING": "Add at least one category to better explain who your publication is geared towards",
```

Comment:
Not sure about the meaning, it's to define the categories? Maybe `Add at least one category to better define the global theme of your content`. Could be improved.